_pytest/nodes.py

```diff
@@ -116,6 +116,7 @@ def ihook(self):
     Function = _CompatProperty("Function")
     File = _CompatProperty("File")
     Item = _CompatProperty("Item")
+    Package = _CompatProperty("Package")
```

Comment:
i missed his earlier, but those properties are scheduled for removal, i'd like to avoid adding more